python/ray/serve/config.py

```diff
@@ -63,10 +64,10 @@ class AutoscalingConfig(BaseModel):
     upscale_delay_s: NonNegativeFloat = 30.0
 
     # Cloudpickled policy definition.
-    serialized_policy_def: bytes = b""
+    _serialized_policy_def: bytes = PrivateAttr(default=b"")
 
     # Custom autoscaling config. Defaults to the request-based autoscaler.
-    policy: Union[str, Callable] = DEFAULT_AUTOSCALING_POLICY
+    _policy: Union[str, Callable] = PrivateAttr(default=DEFAULT_AUTOSCALING_POLICY)
```

Comment:
this doesn't show up in the API reference because it's underscore prefixed right?

Reply:
correct, confirmed here https://anyscale-ray--43291.com.readthedocs.build/en/43291/serve/api/doc/ray.serve.config.AutoscalingConfig.html#ray.serve.config.AutoscalingConfig